quantum/process_keycode/process_auto_shift.c

```diff
@@ -20,10 +20,18 @@
 
 #    include "process_auto_shift.h"
 
-static bool     autoshift_enabled = true;
-static uint16_t autoshift_time    = 0;
+static bool autoshift_enabled = true;
+static uint16_t autoshift_time = 0;
 static uint16_t autoshift_timeout = AUTO_SHIFT_TIMEOUT;
-static uint16_t autoshift_lastkey = KC_NO;
+static int16_t autoshift_lastkey = -1; // -1 is what KC_NO was before - 'not currently autoshifting'.
```

Comment:
Could you revert this?  

Or specifically, run `qmk cformat quantum/process_keycode/process_auto_shift.c` on this file.

Reply:
Done